src/goto-programs/goto_program.h

```diff
@@ -613,6 +634,9 @@ class goto_programt
     instructions.clear();
   }
 
+  /// Get end function target
```

Comment:
Instead of the term `target` it may be better to talk about "instruction iterators?" This applies throughout this PR.